.github/workflows/snyk.yml

```diff
@@ -0,0 +1,31 @@
+name: "Snyk"
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'release-**'
+
+jobs:
+  pom:
+    name: Generate pom.xml for Snyk
+    runs-on: ubuntu-22.04
+    steps:
+
+  monitor:
+    name: Send information to Snyk
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: snyk/actions/setup@master
```

Comment:
can we have some specific version or commit instead of master?

Reply:
I've debated should I do one way or the other, since most of actions use release branches and snyk does not have it... and chose randomly, haha.